### ISSUE TITLE
Loss reaches nan after 300 epochs.

### ISSUE THREAD

**Issue by Aradhya0510** [KinglittleQ/GST-Tacotron#8]
The loss is reaching nan value after 300 epochs(mel and mag both). The generated mel contains infinite value.
Refer this synthesis: https://drive.google.com/file/d/1xRb10hPvLqDTXebjdCi3fAfOAxLyKGss/view?usp=sharing
You can hear the amplitude fluctuations.
Let me know if you've encountered something similar.

**Comment by KinglittleQ** [comment]
Yes, I've encountered such a problem. But I have no idea how to solve it. Try to modify the hyperparameter and train again may solve this problem because this situation won't appear every time.

Furthermore, about 100 epochs are enough to get good results and try to adjust the learning rate after 100 epochs.

Finally, I have to say that my suggestion may not work cause I have a long time not to concern this repo. Sorry about that.

**Comment by Aradhya0510** [comment]
Hey @KinglittleQ I've encountered similar problem with generated mel from Nvidia's tacotron2 implementation, there they've suggested to change the gate threshold value, maybe that'll work for you too. I've gotten bad results with 100 epochs too that's why was trying train more. Anyway do you know any other implementation dealing with expressive tacotron? Thank you!
